src/pages/_app.tsx

```diff
@@ -39,10 +40,25 @@ type AppPropsWithLayout = AppProps & {
   Component: NextPageWithLayout;
 };
 
+const hostUrl =
+  process.env.NODE_ENV === 'development'
+    ? 'localhost:3000'
+    : process.env.VERCEL_URL;
```

Comment:
I'm assuming we use Vercel for deployment but would be good if someone can confirm.  If so, I believe this ENV variable could be a good way to get the current URL (see [here](https://stackoverflow.com/a/66011585/9830506) for more info on this). 

There are a couple other ways we go could with this like just hardcoding `https://veganhacktivists.org` or using a package called [next-absolute-url](https://www.npmjs.com/package/next-absolute-url).

Reply:
We're using digitalocean app-platform. So yeah just use veganhacktivists.org hardcoded